src/_pytest/mark/structures.py

```diff
@@ -47,18 +47,18 @@ def get_empty_parameterset_mark(
 ) -> MarkDecorator:
     from ..nodes import Collector
 
+    argslisting = ", ".join(argnames)
+
     fs, lineno = getfslineno(func)
-    reason = f"got empty parameter set {argnames!r}, function {func.__name__} at {fs}:{lineno}"
+    reason = f"got empty parameter set for ({argslisting})"
```

Comment:
Any reason why you removed the function location?

Reply:
the correct location is already part of the error reporting - so it was giving the name and location twice